metarecord/tests/test_api.py

```diff
@@ -1845,3 +1849,25 @@ def test_bulk_update_delete_permission(bulk_update, user_api_client, permission)
     else:
         assert response.status_code == 204
         assert not BulkUpdate.objects.filter(pk=bulk_update.pk).exists()
+
+
+@pytest.mark.parametrize('permission', (None, 'view_modified_by', 'superuser'))
+@pytest.mark.django_db
+def test_bulk_update_modified_and_created_by_display(bulk_update, user_api_client, permission):
+    if permission == 'view_modified_by':
+        set_permissions(user_api_client, Function.CAN_VIEW_MODIFIED_BY)
+    elif permission == 'superuser':
+        user_api_client.user.is_superuser = True
+        user_api_client.user.save(update_fields=['is_superuser'])
+
+    bulk_update.created_by = user_api_client.user
+    bulk_update.modified_by = user_api_client.user
+    bulk_update.save(update_fields=['created_by', 'modified_by'])
```

Comment:
`created_by` here and in the test's name seem to be a relic from some old code(?)

Reply:
Oops. Good catch. Originally I included `created_by` to the API as well but ended up removing it for consistency because it isn't included in any other endpoints either.